src/rules/font-family-no-duplicate-names/README.md

```diff
@@ -0,0 +1,47 @@
+# font-family-no-duplicate-names
+
+Disallow duplicate font family names.
+
+```css
+a { font-family: "Times New Roman", serif; }
+/**               ↑                 ↑
+ *               These font family names */
+```
+
+This rule checks the `font` and `font-family` properties.
+
+This rule ignores `$sass`, `@less`, and `var(--custom-property)` variable syntaxes.
+
+**Caveat:** This rule does not currently understand escape sequences. If you want to use the font family name "Hawaii 5-0" you will need to wrap it in quotes, instead of escaping it as `Hawaii \35 -0` or `Hawaii\ 5-0`.
```

Comment:
Not using quotes brings up a lot of ambiguity. For example, <pre> `Lucida      Grande`</pre> and <pre>`Lucida Grande`</pre> are one and the same valid family font name from CSS point of view. But this rule will treat these fonts as different font names. On the other hand, <pre>`"Lucida     Grande"`</pre> and <pre>`"Lucida Grande"`</pre> are different font names from both CSS and this rule point of view.